en/docs/how-tos/endpoints/overriding-endpoint-information/overriding-endpoints-for-imported-apis.md

```diff
@@ -3,237 +3,115 @@
 !!! note
     This information is only applicable for APIs imported from WSO2 API Manager.
 
-In the event you change your existing backend service connection URL or add a new backend, you will need to override your backend connection URL that you provided in the respective open API definition. The following section describes how you can override your endpoint information for APIs that are imported from WSO2 API Manager.
+In the event of adding a new backend or changing the existing backend service connection URL, you will need to override your backend connection URL that you provided in the respective open API definition. The following section describes how you can override your endpoint information for APIs that are imported from the WSO2 API Manager.
```

Comment:
In the event, a backend service connection URL is added or edited, the backend connection URL provided in the respective OpenAPI definition needs to be overriden. The following section describes how this can be overriden on the APIs that are imported from the WSO2 API Manager.